### PR TITLE
Mention -native-build-target in README

### PR DESCRIPTION
**Pull request by Zeta611**
and update Changelog.md according to #175

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -136,6 +136,15 @@ This automatically annotates `@live` all the items in the `src/test` and `tmp` f
 -live-paths src/test,tmp
 ```
 
+### CLI -native-build-target
+
+If a native project uses code generation, e.g., via `ocamlyacc`, the build target path needs to be specified to ensure correct analysis.
```
Comment (cristianoc) [inline README.md:141]: Are you sure this is about `ocamlyacc`? I would rather describe the underlying situation, and bring is `ocamlyacc` as an example of that situation.

I guess the situation is: the paths of the files are not relative to the root of the project.
And is `ocamlyacc` the only example of that?

Comment (Zeta611) [reply README.md:141]: Yes, you are correct. I'll update the document.

**Comment by Zeta611** [comment]
Does it sound better now?

**Review by cristianoc — APPROVED** [review]
Thanks!

**Pull request merged**